prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+
+model User {
+  id                  String                  @id @default(uuid())
+  name                String
+  email               String                  @unique
+  password            String
+  created_at          DateTime
+  updated_at          DateTime
+}
+
+model UserAuthProvider {
+  id            String   @id @default(uuid())
+  user_id       String
```

Comment:
Please consider adding [relations](https://www.prisma.io/docs/orm/prisma-schema/data-model/relations) to these models.
Could be something like this:

```suggestion
model User {
  id                  String                  @id @default(uuid())
  name                String
  email               String                  @unique
  password            String
  created_at          DateTime
  updated_at          DateTime
  posts UserAuthProvider[]
}

model UserAuthProvider {
  id            String   @id @default(uuid())
  user        User @relation(fields: [user_id], references: [id])
  user_id       String
  provider      String
  provider_user String
  created_at    DateTime
}
```